pkg/server/test/read_changes.go

```diff
@@ -123,7 +122,7 @@ func TestReadChanges(t *testing.T, datastore storage.OpenFGADatastore) {
 			},
 		}
 
-		readChangesQuery := commands.NewReadChangesQuery(backend, logger.NewNoopLogger(), encoder, 0)
+		readChangesQuery := commands.NewReadChangesQuery(backend, commands.WithReadChangesQueryEncoder(encoder))
```

Comment:
This `encoder` is actually an `encrypter`. We don't need this for testing. Can you remove these lines above?

```go
encrypter, err := encrypter.NewGCMEncrypter("key")
require.NoError(t, err)

encoder := encoder.NewTokenEncoder(encrypter, encoder.NewBase64Encoder())
```

Let's just just the default base 64 encoder in tests :)